### PR TITLE
Add Wisdem to conda-forge

### PR DESCRIPTION
**Pull request by jefalon**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/wisdem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - cython
+    - numpy
+    - setuptools
+    - scipy
+    - make
+    - compilers  # [not win]
```
[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/wisdem/meta.yaml]: use `{{ compiler('c') }}` in the build sections

[the remark this answers is not in the document]
Comment (jefalon) [reply recipes/wisdem/meta.yaml]: Thanks for helping me with that! I moved the compilers to the build section. Hopefully that works now.

File: recipes/wisdem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - python >=3.7
+    - numpy
+    - setuptools
+    - cython
+    - scipy
+  run:
+    - python >=3.7
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - matplotlib
```
Comment (xhochy) [inline recipes/wisdem/meta.yaml]: depend on `matplotlib-base`. `matplotlib` is an end user convenience package of `matplotlib-base`and the Qt backend.
```suggestion
    - matplotlib-base
```

**Comment by jefalon** [comment]
Hey @scopatz and @xhochy, 

I was finally able to get the wisdem recipe to build on all systems. I was wondering if you could take look to see if wisdem is ready to be added to conda forge? 

Thanks,

Jeff

**Comment by scopatz** [comment]
Thanks!

**Comment by jefalon** [comment]
Hey @scopatz, Thanks for adding wisdem to conda-forge! Do you know why only the Windows build is available?

**Comment by scopatz** [comment]
It might have failed on the feedstock

**Comment by scopatz** [comment]
running build now: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=74982

**Comment by jefalon** [comment]
Fantastic! Everything seems to be working now. Thank you so much!

**Comment by scopatz** [comment]
Excellent!

**Pull request merged**